### PR TITLE
FIX: presentation styles file

### PR DESCRIPTION
**Pull request by wonecode**

### CODE REVIEW

**Review by jaldabaoth-code — DISMISSED** [review]

File: templates/home/index.html.twig [hunk 1]
```diff
@@ -1,88 +1,162 @@
...
-				<p class="mb-0 txt-banner">Temps moyen Aller/Retour Domicile/Travail</p>
-			</div>
-			<div class="col card-banner text-center py-3 mx-3">
-				<i class="fas fa-map-marked-alt mb-3"></i>
-				<p class="title-icons">3/4 Français</p>
-				<p class="mb-0 txt-banner">Prêts à changer de travail pour se rapprocher de leur domicile</p>
-			</div>
-		</div>
-	</header>
-    <section class = "freeMap">
```
Comment (RaphaelBS-WCS) [inline templates/home/index.html.twig:42]: You have removed this, but it's the map

File: templates/home/index.html.twig [hunk 2]
```diff
@@ -1,88 +1,162 @@
...
+            </div>
+            <div class="col card-banner text-center py-3 mx-3">
+                <i class="fas fa-map-marked-alt mb-3"></i>
+                <p class="title-icons">3/4 Français</p>
+                <p class="mb-0 txt-banner">Prêts à changer de travail pour se rapprocher de leur domicile</p>
+            </div>
+        </div>
+    </header>
+
+    <section>
```
Comment (RaphaelBS-WCS) [inline templates/home/index.html.twig]: In the PR which support the fix, we had an id = "presentation" to this section, to point an anchor to the "Découvrir JobPermut" Button. Would you add it please?

**Review by RaphaelBS-WCS — CHANGES_REQUESTED** [review]
See comments

**Review by RaphaelBS-WCS — DISMISSED** [review]
Approved

**Review by RedPandore — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**